bin/ruby-build

```diff
@@ -918,6 +919,28 @@ use_homebrew_yaml() {
   fi
 }
 
+use_freebsd_pkg() {
+  # check if FreeBSD
+  if [ "FreeBSD" = "$(uname -s)" ]; then
+      # use openssl if installed from Ports Collection
+      if [ -f /usr/local/include/openssl/ssl.h ]; then
+	  package_option ruby configure --with-openssl-dir="/usr/local"
+      fi
+
+      # check if 11-R or later
+      if [ "$(uname -r | sed 's/[^[:digit:]].*//')" -ge 11 ]; then
```

Comment:
```uname -r``` returns like ```11.0-RELEASE-p2``` string.
This includes major and minor release number.
For comparing with ```-ge```, numbers must be integer.
I will fix to cut off dot and following characters.